packages/backend/src/queryBuilder.ts

```diff
@@ -691,7 +691,7 @@ export const buildQuery = ({
             sqlSelect,
             sqlFrom,
             sqlJoins,
-            customDimensionSql
```

Comment:
Like in here : https://github.com/lightdash/lightdash/pull/8010/files#diff-641ce89defd0e168c1fcf517d68dbfaecbcdd76f6063158efae1d2826a086ac1L737